test/unit/spec/webauthn_spec.js

```diff
@@ -72,7 +71,7 @@ function (Q, Expect, webauthn) {
       expect(webauthn.isAvailable()).toBe(true);
     });
 
-    it('makeCredential response has xhr.responseJSON.errorSummary if NotSupportedError is triggered',
+    /*it('makeCredential response has xhr.responseJSON.errorSummary if NotSupportedError is triggered',
```

Comment:
Same with this one. In general, I see that there are no tests added - add tests for the functionality you add.